ui/public/i18n/language.json

```
@@ -30,7 +30,9 @@
     "sftpgo_admin_login_tips": "The default login:password is admin:admin, you must change it at the first login",
     "sftpgo_user_login_tips": "The credentials can be found in the virtualhost page",
     "sftp_tcp_port_tips": "Use SFTPGo or connect a sftp client: sftp -P {port} user@hostname",
-    "sftpgo_upload_website":"Upload your websites"
+    "sftpgo_upload_website":"Upload your websites",
+    "sftpgo_not_running": "sftpgo is not configured",
```

```suggestion
    "sftpgo_not_running": "SFTPGo is not configured",
```
Just a minor suggestion to use the correct case for "SFTPGo"